go.mod

```diff
@@ -1,71 +1,108 @@
 module github.com/aws/shim-loggers-for-containerd
 
 require (
+	github.com/aws/aws-sdk-go-v2 v1.21.0
+	github.com/aws/aws-sdk-go-v2/config v1.18.39
+	github.com/aws/aws-sdk-go-v2/service/cloudwatchlogs v1.23.5
 	github.com/cihub/seelog v0.0.0-20170130134532-f561c5e57575
 	github.com/containerd/containerd v1.5.18
```

Comment:
The install-containerd script will use the containerd version in go.mod, so it might be good to update this to 1.6 version.